test/benchmarks/material/checkbox/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
+
+# todo(wagnermaciel): If assets are defined and no index.html is provided, throw error with clear error message.
```

Comment:
Wouldn't this TODO be for the `component_benchmark` rule rather than this invocation of it? Should this say something like this?
```
TODO(wagnermaciel): Update this target to use ___ once `component_benchmark` supports _____.
```

Reply:
@jelbourn We might be better off just creating an issue in `angular/angular`. That way we can centralize the discussion on changes to the `component_benchmark` rule. What are your thoughts?

Reply:
You can file an issue there, but you'll also be the only one looking at it :)

I just want to make sure we don't lose track of this